solution/index.js

```diff
@@ -0,0 +1,554 @@
+// ============================
+// ======== Consts ============
+// ============================
+const KEY_1 = 49;
+const KEY_2 = 50;
+const KEY_3 = 51;
+const BACKSPCE_KEYCODE = 8;
+
+//==========================
+// ===== Global Vars =======
+//==========================
+
+let editTextValue = "";
+let searchInput = document.querySelector("#search");
+let pressed = new Set();
+let errorLabel = document.querySelector("#error-label");
+let contextMenu = document.querySelector("#context-menu");
+let SelectedTask;
+let SelectedTaskName;
+let draggedTask;
+let editFlag = false;
+
+let addToDoButton = document.querySelector("#submit-add-to-do", );
+let addInProgressButton = document.querySelector("#submit-add-in-progress");
+let addDoneButton = document.querySelector("#submit-add-done");
+let saveButton = document.querySelector("#save-btn");
+let loadButton = document.querySelector("#load-btn");
+
+// ============================
+// ====== Main run ============
+// ============================
+
+addToDoButton.addEventListener('click', onAddClickHandler);
+addInProgressButton.addEventListener('click', onAddClickHandler);
+addDoneButton.addEventListener('click', onAddClickHandler);
+saveButton.addEventListener('click', onSaveClickHandler);
+loadButton.addEventListener('click', onLoadClickHandler);
+searchInput.addEventListener('keyup', onKeyUpHandler);
+
+document.addEventListener('click', onScreenClick);
+document.addEventListener("dragover", onTaskDragOver);
```

Comment:
Do not add an event to the document level. 
the 'click' event above is working but thats not very good logic.